advanced/index.md

```diff
@@ -71,15 +71,15 @@ Historical commit points are simple to discuss using *commitish* rather than hex
 Shorthand              | Explanation
 ---------------------- | -----------
 `HEAD`                 | Current commit
-`HEAD^^`               | Second parent of current commit
-`HEAD~2`               | Two commits from current commit
+`HEAD^2`               | Second parent of current commit (assumes current commit the result of a merge)
+`HEAD~2`               | First parent (commit) of the first parent of the current commit (different than above!)
```

Comment:
Happy to see tune-ups on these descriptions happening. If we'd like to be consistent with Git documentation, and even the Pro Git book, I'd suggest we use the same vocabulary from those docs here as well.

Suggestions:

> `HEAD^2` Second parent if a merge commit, otherwise second ancestor.

> `HEAD~2` Second ancestor of the current commit